Database/Database.sql

```diff
@@ -0,0 +1,344 @@
+/* CREATE DATABASE */
+DROP DATABASE IF EXISTS suitapp_db;
+CREATE DATABASE suitapp_db;
+\c suitapp_db;
+
+/* CREATE TABLES */
+CREATE TABLE users
+(
+    id SERIAL PRIMARY KEY,
+    firstname VARCHAR (20) NOT NULL,
+    lastname VARCHAR (20) NOT NULL,
+    username VARCHAR (20) UNIQUE NOT NULL,
+    user_password VARCHAR (30) NOT NULL,
+    email VARCHAR (30) UNIQUE NOT NULL,
+    avatar_url TEXT,
+    bio VARCHAR (500),
+    ui_theme BOOLEAN NOT NULL DEFAULT TRUE,
```

Comment:
I will suggest naming this even `dark_theme` or `light_theme` to go along with the boolean values it holds.
It's just an opinion

Reply:
I like that idea - it makes it explicit what's being turned on or off. I'll make a change to `light_theme` so it defaults to the light theme of our site.